packages/gguf/tsconfig.json

```diff
@@ -13,6 +13,6 @@
 		"noImplicitOverride": true,
 		"outDir": "./dist"
 	},
-	"include": ["src"],
+	"include": ["src", "../shared/src"],
```

Comment:
```suggestion
	"include": ["src"],
```

Reply:
Resolved in https://github.com/huggingface/huggingface.js/pull/656/commits/1caedf34106013a2e1f7527d6a1db890da6eb2d6